python/GafferSceneUI/CameraUI.py

```diff
@@ -276,17 +474,25 @@
 
 			"description",
 			"""
-			To use fStop to compute the lens aperture, we need to know focal length in world units.
-			Since we usually store focal length in millimeters to match how we refer to real focal lengths
-			and apertures, we need to know how to scale focal length into world units.
-
-			We default to a value of 0.1, which scales from millimeters to centimeters.  This matches
-			the default world units of Alembic and USD.  If your world units are decimeters or meters,
-			then pick the corresponding option instead.
-
-			If you are controlling the camera using a field of view instead of focal length, then the
-			default aperture is just 1.  You should pick "Custom" and then pick a scale that matches
-			a realistic aperture size ( ie. 0.036 meters )
+			The scale to convert from focal length units to world space
+			units. Combined with f-stop to calculate the lens aperture.
+
+			For example, take a 50mm focal length, and a scene that
+			uses decimeters (dm) for its world space units. The
+			mm-to-dm conversion would be: `50mm × 0.01 = 0.05dm`.
+
+			The default value of 0.1 scales millimeter (default	focal
+			length unit) to centimeter (default world space unit of
+			Alembic and USD scene formats). Other default settings for
+			scaling to decimeter or meter are also available.
+
+			If using _Field Of View_ projection mode, you won't have a
+			focal length plug to work with, and the aperture size will
+			be (1,1). To compensate, select _Custom_ and then input a
+			value that scales the scene unit of measure to a realistic
+			aperture size. For example, 0.35 would convert 1 centimeter
+			(Alembic/USD default) to 35mm, which would simulate a 35mm
```

Comment:
The last paragraph was meant more as a tip, because _Field of View_ mode uses an aperture of 1x1mm, which isn't obvious unless you look at the camera's parameters in the inspector.

Reply:
Sorry, my comment wasn't so much about having a 'tip', more about switching to talking about focal length rather than aperture. @danieldresser do you have any thoughts?